exonum-java-binding/core/src/main/java/com/exonum/binding/core/service/Service.java

```diff
@@ -98,6 +98,21 @@ default void configure(Fork fork, Configuration configuration) {
    */
   void createPublicApiHandlers(Node node, Router router);
 
+  /**
+   * Handles the changes made by all transactions included in the upcoming block.
```

Comment:
@alekseysidorov would you check please if the user-facing documentation is accurate?